robot/marker_setup/markers.py

```diff
@@ -0,0 +1,110 @@
+import enum
+import typing
+
+from .teams import TEAM
+
+
+class MARKER_TYPE(enum.Enum):
+    POTATO = enum.auto()
+    ARENA = enum.auto()
+
+
+# class MARKER_OWNER(enum.Enum):
+#     ME = enum.auto()
+#     ARENA = enum.auto()
+#     ANOTHER_TEAM = enum.auto()
```

Comment:
Just thinking about future generations of robotics people. If this is not needed then lets delete it so there is less clutter for them